test/integration/http/http_test.go

```diff
@@ -38,13 +39,23 @@ func TestHttp(t *testing.T) {
 }
 
 func TestHttpJob(t *testing.T) {
+	testHttpJob(t, "http://httpbin:8080/")
+	testHttpJob(t, "http://nghttp2:8443/")
```

Comment:
Is this second URL supposed to be here or moved inside `TestHttp2Job`?

Reply:
yup! but will be removed, good catch.